pkg/operator/k8sutil/events_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package k8sutil for Kubernetes helpers.
+package k8sutil
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
+)
+
+func TestReport(t *testing.T) {
+
+	pod := &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "test-pod",
+		},
+	}
+	eventType, eventReason, eventMsg := "Normal", "Created", "Pod has been created"
+
+	eventKey, err := getEventKey(pod, eventType, eventReason, eventMsg)
+	assert.NoError(t, err)
+
+	reporter := NewEventReporter(record.NewFakeRecorder(1024), 5, 1)
```

Comment:
Can we change this to `NewEventReporter(record.NewFakeRecorder(1024), 1, 15)` or something like that, 5 events per minute is too much IMO. 
Justification: I'd say that we'd ideally want events to repeat only as often as the apiserver deletes them, the default is an hour, but it can be configured as an api-server flag, so a 15 minutes sounds like a conservatively low number.

Reply:
If this is just for the test code I don't see a need for a change.